api/base/schemas/utils.py

```diff
@@ -26,13 +26,18 @@ def validate_user_json(value, json_schema):
 
 
 def validate_dates(history):
+    now = datetime.now()
```

Comment:
I think this implementation that allows for a future `end_date` and for `ongoing` to be true even if an `end_date` is provided is better than my solution for this issue (https://github.com/CenterForOpenScience/osf.io/pull/9163/files) which does not acknowledge a future `end_date`.

Reply:
Drive by: The current presumption is that, if ongoing is checked, `end_date` is empty/null/None.

Reply:
I think I did this to satisfy an existing test that supplied an end date with ongoing set to True (might have been a typo). Upon further consideration this has the potential to create some nasty data, so lets not do this.